test/NuGet.Clients.Tests/NuGet.PackageManagement.UI.Test/Actions/AccessiblePackageIdentityTests.cs

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using NuGet.Packaging.Core;
+using NuGet.Versioning;
+using Xunit;
+
+namespace NuGet.PackageManagement.UI.Test
+{
+    public class AccessiblePackageIdentityTests
+    {
+        [Fact]
+        public void AutomationName_WhenCultureIsNeutral_ReturnsMessage()
+        {
+            var version = new NuGetVersion(0, 0, 1);
+            var result = new AccessiblePackageIdentity(new PackageIdentity("test.package", version));
+
+            Assert.Equal("test.package version 0.0.1", result.AutomationName);
```

Comment:
Why do you have 2 very similar asserts.

Reply:
One is for detecting the most simple case. The second is to detect , in a case of a test failure, where the possible error is.

Reply:
I don't think I follow. 

You would get the same error message in both cases.